components/automate-gateway/gateway/services.go

```diff
@@ -427,7 +427,7 @@ func (s *Server) ProfileCreateHandler(w http.ResponseWriter, r *http.Request) {
 	var cType, profileName, profileVersion string
 	contentTypeString := strings.Split(r.Header.Get("Content-type"), ";")
 	switch contentTypeString[0] {
-	case "application/json":
+	case "application/json", "application/json+lax":
 		cType = r.Header.Get("Content-type")
```

Comment:
If we do
```suggestion
		cType = "application/json"
```

here, can't we *not* change https://github.com/chef/automate/pull/3752/files#diff-09082aecbe7f4ed3241f06fc08a1afe2R153?

Reply:
If we ever get to refactoring this: the downstream service could take a different message depending on what automate-gateway decides (based on the content-type). What we do now -- passing along the content-type, and having both payload types (supermarket profile / profile upload) in one message type -- doesn't seem like the most obvious solution to  me 😉 (Happy to chat, of course, too.)